ui/src/plugins/en.json

```diff
@@ -82,13 +82,14 @@
   "absenteeRequest": "Request an absentee ballot",
   "absenteeImageAlt": "stock image of voters at a voting poll",
   "absenteeRequested": "Already requested ballot or planning to vote early?",
-  "absenteeMoreInfo": "More information from your state about absentee ballots",
+  "absenteeMoreInfo": "[More information]({moreUrl}) from your state about absentee ballots.",
```

Comment:
consider rewording to keep to the singular "absentee ballot" usage, based on the rest of the use of the term on the page where this is the only pluralized form I saw.


(i don't have a suggestion myself though or I would provide it...)